packages/taquito-sapling/src/sapling-tx-builder/sapling-module-wrapper.ts

```diff
@@ -0,0 +1,146 @@
+import * as sapling from '@airgap/sapling-wasm';
+import { randomBytes } from '@stablelib/random';
+import {
+  ParametersOutputProof,
+  ParametersSpendProof,
+  ParametersSpendSig,
+  SaplingTransactionInput,
+} from '../types';
+import * as path from 'path';
+import * as fs from 'fs';
+import axios, { AxiosResponse } from 'axios';
+
+const SAPLING_PARAMS_DIR = path.resolve(__dirname, 'sapling-params');
+const ZCASH_DOWNLOAD_URL = 'https://download.z.cash/downloads';
+const SPEND_PARAMS_FILE_NAME = 'sapling-spend.params';
+const OUTPUT_PARAMS_FILE_NAME = 'sapling-output.params';
```

Comment:
are these variables something that are constant? Shall we consider moving them to `constants`?